deb/debian/control.in

```diff
@@ -388,18 +386,6 @@ Description: Metapackage to setup and configure a "Master Backend" profile of My
  installed on a server rather then a desktop. It makes no sense to install it
  on anything but the first backend machine on the network.
 
-Package: mythtv-dbg
-Architecture: any
-Section: debug
-Priority: extra
-Recommends: libc6-dbg
-Conflicts: mythplugins-dbg
-Replaces: mythplugins-dbg
-Depends: ${misc:Depends}, mythtv-backend (=${binary:Version}) | mythtv-frontend (=${binary:Version})
-Description: Debug symbols for mythtv packages
- This package provides all debug symbols for mythtv packages since ddebs
- don't appear to always do the right thing.
-
```

Comment:
The actual reason for this package was so that it made bug reports from users who use the Ubuntu PPA higher quality.  At least at the time debug symbols were not not kicked into ddebs.  Also users weren't able to replicate problems sometimes so this made the report the first time higher quality.

So I would prefer that we don't have this done in the short term at least.

Reply:
As the git comment of b7b98c04b7ff6b72d8f03f571a2557d7300e0570 describes Debian automatically build debug packages - I don't know if that also work with PPAs but at least this feature was coordinated between Debian and Ubuntu.
The short summary is that you no longer need to **explicitly** create debug packages **manually**, but you always get them for free **automatically**: The defaults for the compiler was changed to always built libraries and binaries with debug symbols enabled, and then `dh_strip` separates those symbols from the libraries/binaries and moves them to a separate package with the suffix `-dbgsym`. You don't need to (actually must not) specify them in the `debian/rules` file, they are completely automatic:
```
# ls -1 *-dbgsym_*.deb
libmyth-dbgsym_31.0+fixes.20200326.9579662cdc-0ubuntu0_amd64.deb
mytharchive-dbgsym_31.0+fixes.20200326.9579662cdc-0ubuntu0_amd64.deb
mythbrowser-dbgsym_31.0+fixes.20200326.9579662cdc-0ubuntu0_amd64.deb
mythgame-dbgsym_31.0+fixes.20200326.9579662cdc-0ubuntu0_amd64.deb
mythmusic-dbgsym_31.0+fixes.20200326.9579662cdc-0ubuntu0_amd64.deb
mythnews-dbgsym_31.0+fixes.20200326.9579662cdc-0ubuntu0_amd64.deb
mythtv-backend-dbgsym_31.0+fixes.20200326.9579662cdc-0ubuntu0_amd64.deb
mythtv-common-dbgsym_31.0+fixes.20200326.9579662cdc-0ubuntu0_amd64.deb
mythtv-frontend-dbgsym_31.0+fixes.20200326.9579662cdc-0ubuntu0_amd64.deb
mythtv-transcode-utils-dbgsym_31.0+fixes.20200326.9579662cdc-0ubuntu0_amd64.deb
mythweather-dbgsym_31.0+fixes.20200326.9579662cdc-0ubuntu0_amd64.deb
mythzoneminder-dbgsym_31.0+fixes.20200326.9579662cdc-0ubuntu0_amd64.deb
```
As those debug information are quiet large, are used by very few people (mostly developers) and storage for the Debian repositories are provided by many organizations world-wide for free, those debug packages are not hosted in the main Debian repository but in a separate section `/debian-debug/`, so mirror operators can easily op-out from mirroring them.
So my commit removes the explicit debug package, but you don't loose that information.

I can understand you wish to get useful error reports. I haven't checked if MythTV itself contains a SIGSEGV handler to automatically create a traceback on crashes; that one would definitely need those debug information as it runs on the users system itself. If I remember correctly (main) Ubuntu has some infrastructure ([apport](https://wiki.ubuntuusers.de/Apport/)) to automatically get them and Canoncial has some infrastructure post-process those reports and augment those crash reports with debug information from those packages, but I don't know if you also can use this infrastructure for your PPAs.

One more issue might be updates of old MythTV setups where this old `mythtv-dbg` package was installed (by default): As the package declares an exact match on the corresponding `mythtv` package APT will refuse to upgrade `mythtv` as that would break the same-version requirement. The right solution would be to convert the `mythtv-dbg` package into a transitional package depending on the new `*-dbgsym` packages for some time and only remove it in the future when hopefully each setup out there at least upgraded to the transitional package once.